src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -164,9 +168,16 @@
 
     private GenomeLocSortedSet genomeLocSortedSet;
 
+    @Override
     public void onTraversalStart() {
-        if (VALIDATE_GVCF)
-            genomeLocSortedSet = new GenomeLocSortedSet(new GenomeLocParser(getBestAvailableSequenceDictionary()));
+        if (VALIDATE_GVCF) {
+            SAMSequenceDictionary dict = getBestAvailableSequenceDictionary();
+
+            if (dict == null)
+                throw new UserException("Could not find valid sequence dictionary for this command.");
```

Comment:
This would be more useful if it was a bit more specific I think.  ex:  something like: "Validating a GVCF requires a sequence dictionary but no dictionary was found in your input."